contributing.md

```diff
@@ -0,0 +1,37 @@
+# What this repository is
+This repository contains sources of the Introduction to Python course.
```

Comment:
This repository contains the source material for the Introduction to Python course.